common/beerocks/bwl/shared/nl80211_client_impl.h

```diff
@@ -42,6 +42,18 @@ class nl80211_client_impl : public nl80211_client {
      */
     virtual ~nl80211_client_impl() = default;
 
+    /**
+     * @brief Gets radio information.
+     *
+     * Radio information contains HT/VHT capabilities and the list of supported channels.
+     *
+     * @param[in] interface_name Interface name, either radio or Virtual AP (VAP).
+     * @param[out] radio_info Radio information.
+     *
+     * @return True on success and false otherwise.
+     */
+    virtual bool get_radio_info(const std::string &interface_name, sRadioInfo &radio_info) override;
```

Comment:
Same comment, no need for virtual here (there are a lot of places where it's done like this, but also others where we drop the override)

Reply:
I think we didn't come to a conclusion about it in the cppcheck PR. Vote in #democracy?